client/src/components/main_content/card/season/CardSeason.jsx

```diff
@@ -63,53 +57,55 @@ const CardSeason = ({
 		);
 	}
 	if (reviews) {
-		const vote_average =
-			reviews.author_details.rating &&
-			reviews.author_details.rating;
-
+		const regex = /https:\/\/\S+/g;
+		const path = reviews?.author_details?.avatar_path;
+		const avatarUrl = path?.match(regex);
+		const avatar = `${img_avatar}/${path}`;
+		const air_date = dayjs(reviews?.created_at).format(
+			'MMM D, YYYY',
+		);
+		const vote_average = reviews.author_details.rating
+			? reviews.author_details.rating
+			: '0';
 		return (
 			<div className='wrap_card cardSeason cardReviews'>
 				<Link to={link}>
 					<div className='wrap_poster reviewsAvatar'>
-						{_media_type === 'tv' ? (
-							<img
-								src={
-									reviews?.author_details?.avatar_path
-										? `${img_avatar}/${reviews?.author_details?.avatar_path}`
-										: noProfile
-								}
-								alt='Avatar'
-							/>
-						) : (
-							<img
-								src={
-									reviews?.author_details?.avatar_path
-										? reviews?.author_details?.avatar_path.substring(
-												1,
-										  )
-										: noProfile
-								}
-								alt='Avatar'
-							/>
-						)}
+						<img
+							src={path ? avatarUrl || avatar : noProfile}
+							alt='Avatar'
+						/>
 					</div>
 				</Link>
 				<div className='des'>
-					<div className='title titleSeason titleReviews'>
+					<div className='titleReviews'>
 						<Link
-							to={link}
+							to={`/review/${reviews?.id}`}
 							style={{
 								textDecoration: 'none',
 								color: '#fff',
 							}}>
 							<h4>{`A review by ${reviews?.author}`}</h4>
 						</Link>
 						<p className='wrap_vote'>
-							<AiFillStar className='icon_star' />
+							<AiFillStar
+								className='icon_star'
+								color={
+									vote_average >= 9.0
+										? '#57e32c'
+										: vote_average <= 8 && vote_average >= 6
+										? '#ffe234'
+										: vote_average <= 5 && vote_average >= 3
+										? '#ffa534'
+										: vote_average <= 2
+										? '#ff4545'
+										: ''
+								}
```

Comment:
ini klo di buat function bgimn? jd nnti tinggal functionya mi sj yg di panggil
```suggestion
								color={
									vote_average >= 9.0
										? '#57e32c'
										: vote_average <= 8 && vote_average >= 6
										? '#ffe234'
										: vote_average <= 5 && vote_average >= 3
										? '#ffa534'
										: vote_average <= 2
										? '#ff4545'
										: ''
								}
```
```suggestion
								color={
									vote_average >= 9.0
										? '#57e32c'
										: vote_average <= 8 && vote_average >= 6
										? '#ffe234'
										: vote_average <= 5 && vote_average >= 3
										? '#ffa534'
										: vote_average <= 2
										? '#ff4545'
										: ''
								}
```